ui_patterns.info.yml

```diff
@@ -2,6 +2,5 @@ name: UI Patterns
 type: module
 description: UI patterns
 core: 8.x
+core_version_requirement: ^8 || ^9
 package: User interface
-dependencies:
-  - drupal:system (>=8.3.0)
```

Comment:
We should keep this if possible, would this work?
```yml
 core_version_requirement: >=8.3.0 || ^9
```

Reply:
We can set `core_version_requirement: ^8.3 || ^9` but I'm not sure that there is sense due to `core_version_requirement` has been introduced in  Drupal 8.7.7. So all versions of Drupal lower this don't read such parameter. Also supported versions of Drupal are >=8.8.

Reply:
If I may jump in, the Drupal Update helper bot always suggests having `^8` for the Drupal 8 part of the requirement. As YenYasinn mentioned above, older versions of Drupal 8 won't be able to install recent versions of the module because they won't read the `core_version_requirement` and they won't have `core` to fall back on. So, only sites running Drupal 8.8 and above will be able to install the latest version of the module.

Reply:
Yep, ok we can skip this.